x-pack/libbeat/management/blacklist.go

```diff
@@ -52,7 +51,7 @@ func NewConfigBlacklist(cfg ConfigBlacklistSettings) (*ConfigBlacklist, error) {
 	for field, pattern := range cfg.Patterns {
 		exp, err := match.Compile(pattern)
 		if err != nil {
-			return nil, errors.Wrap(err, fmt.Sprintf("Given expression is not a valid regexp: %s", pattern))
+			return nil, fmt.Errorf("Given expression is not a valid regexp: %s", pattern)
```

Comment:
I know this was existing (and in a lot of other places in the PR), but I would prefer to see us change the error messages to start with [lowercase](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) if we are already touching the line.